app/components/post_body/post_body.js

```diff
@@ -390,14 +390,26 @@ export default class PostBody extends PureComponent {
         } else if (postType === Posts.POST_TYPES.COMBINED_USER_ACTIVITY) {
             const {allUserIds, allUsernames, messageData} = postProps.user_activity;
             messageComponent = (
-                <CombinedSystemMessage
+                <SystemMessage
```

Comment:
Why render a `SystemMessage` instead of a `CombinedJoinLeave` message here?

Reply:
Did we not discuss encapsulating different types of messages under a single uniform component in comments on this very PR? Am I confused?

Reply:
The thing that's confusing me is that we have two cases here that both have basically the same result, and then the SystemMessage component also checks for those same cases. I'd be fine if either one of the `PostBody` or `SystemMessage` components were responsible for identifying combined user activity messages, but right now, both components have to know about them